pyproject.toml

```diff
@@ -83,3 +83,9 @@ reports = true
 # Disable len-as-condition since len(lst) == 0 is more clear on types
 # where as if <var> is confusing since it's not clear what <var> is.
 disable = ["no-else-return", "len-as-condition"]
+
+[tool.pyright]
+exclude = ["doc/**", "e2e/**", "test_scripts/**", "hammer/technology/asap7/gds_scale.py"]
```

Comment:
Is gds_scale.py excluded due to the conditional gdstk/gdspy imports?

Reply:
I removed this exclusion and added one type ascription to the `gds_scale.py`, which seems to satisfy pyright